app/src/main/res/values/strings.xml

```diff
@@ -51,10 +51,10 @@
     <string name="developer_options_ua_vr">VR</string>
     <string name="developer_options_reset">Reset Developer Settings</string>
     <string name="developer_options_reset_button">Reset</string>
-    <string name="developer_options_env">Environment</string>
-    <string name="developer_options_env_meadow">Meadow</string>
-    <string name="developer_options_env_cave">Cave</string>
-    <string name="developer_options_env_void">Void</string>
+    <string name="developer_options_environments">Environment</string>
+    <string name="developer_options_env_meadow">meadow</string>
```

Comment:
why ar these lowercased but the `Pointer Color` selections, for example, are not?